test/pantry-test.js

```diff
@@ -13,18 +15,82 @@ describe('Pantry', function() {
 
   beforeEach(function() {
     userInfo = data[0];
-    user = new User(userInfo)
+    user = new User(userInfo);
+    pantry = new Pantry(user);
+
+    recipe1 = {name: 'Flour Soda', type: ['italian', 'dinner'], ingredients: [{
+      "name": "all purpose flour",
+      "id": 20081,
+      "quantity": {
+        "amount": 1.5,
+        "unit": "c"
+      }
+    },
+    {
+      "name": "baking soda",
+      "id": 18372,
+      "quantity": {
+        "amount": 0.5,
+        "unit": "tsp"
+      }
+    }]};
+
+    recipe2 = {name: 'Cannot make this', type: ['italian', 'dinner'], ingredients: [{
+      "name": "sharks",
+      "id": 8585858585,
+      "quantity": {
+        "amount": 1.5,
+        "unit": "c"
+      }
+    },
+    {
+      "name": "lizards",
+      "id": 18372001001,
+      "quantity": {
+        "amount": 0.5,
+        "unit": "tsp"
+      }
+    }]};
+
 
-    recipe = {name: 'Chicken Parm', type: ['italian', 'dinner']};
   });
 
   it('should be a function', function() {
```

Comment:
- this test on line 58 will not work as intended as written
- it is comparing two complex pieces of data with `eq`
  - we will go back in and readjust this to be suitable
    - may be better to compare directly to the json objects
    - will think on this

Reply:
Apologies. This is actually the test on line 64.

For now, I have made this `(pantry.contents).to.deep.equal(user.pantry)`